B/booster_jl/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "booster_jl"
+version = v"0.1.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/evolbioinfo/booster.git", "a5f5044b192db7ab5572f44ef46abaea007f5ce9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+mkdir -p ${WORKSPACE}/builddir
+cd ${WORKSPACE}/builddir/
+cp -r ${WORKSPACE}/srcdir/booster/* .
+cd src/
+make;
+install -D booster ${prefix}/bin/booster
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("riscv64", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "freebsd"; ),
+    Platform("aarch64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
+
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("booster", :booster)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
```

Comment:
https://buildkite.com/julialang/yggdrasil/builds/21880#0197f0fd-6c73-4564-a150-45fced13c7c7/699-813
```
┌ Warning: To ensure that the correct version of libgomp is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /cache/julia-buildkite-plugin/depots/e2fd9734-29d8-45cd-b0eb-59f7104f3131/packages/BinaryBuilder/zpeEH/src/auditor/compiler_abi.jl:5
```
```suggestion
dependencies = [
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae")),
```
I'm going to guess you'll need LLVM OpenMP for macOS.